docker-compose.yaml

```diff
@@ -0,0 +1,55 @@
+version: '3.8'  
+
+services:
+  postgres: 
+    image: postgres:latest 
+    container_name: postgres_db
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: postgres      
+      POSTGRES_PASSWORD: password  
+      POSTGRES_DB: maxitdb          
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    networks:
```

Comment:
You don't need to specify network in docker compose, since all the containers run in the same network